### ISSUE TITLE
Color CC: Add RGBW value ID

### ISSUE THREAD

**Issue by AlCalzone** [zwave-js/zwave-js#806]

**Comment by robertsLando** [comment]
What do we need for this?

**Comment by AlCalzone** [comment]
We need to which format the hex string should have - which digits, with or without `#`?. OZW concatenates a bunch of HEX digits:
https://github.com/OpenZWave/open-zwave/blob/master/cpp/src/command_classes/Color.cpp#L210

I'm not sure if its worth going past `#RRGGBB` with this, since it is standard and everything else is missing discoverability (maybe `WW`, but that could be warm or cold white...).
We also need to figure out what to do with devices that don't support one of the basic colors. Leave it out, ignore it?

Basically what needs to be done is:
* Probably during the interview, metadata for this value ID should be created
* When a color report is received, the new value of this string must be figured out (from existing values and the new value) and also written
* When this value is set (via the SET_VALUE API) it must be split into its parts to call the set method.

Hope I didn't forget anything.

**Comment by robertsLando** [comment]
> When a color report is received, the new value of this string must be figured out (from existing values and the new value) and also written

And reported to the user by an event :)

**Comment by AlCalzone** [comment]
> And reported to the user by an event :)

That happens when you set a value in the value DB
